translate/src/modules/entitydetails/components/Metadata.tsx

```diff
@@ -86,6 +86,44 @@ function GroupComment({ comment }: { comment: string }) {
   );
 }
 
+function EntityCreatedDate({ dateCreated }: { dateCreated: string }) {
+  // Create date and time formatters
+  const dateFormatter = new Intl.DateTimeFormat('en-US', {
+    day: 'numeric',
+    month: 'long',
+    year: 'numeric',
+  });
+  const timeFormatter = new Intl.DateTimeFormat('en-US', {
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+  });
+
+  // Create a Date object from the dateCreated string
+  const date = new Date(dateCreated);
+
+  // Check if date is a valid Date object
+  if (isNaN(date.getTime())) {
+    console.error(
+      `EntityCreatedDate component received invalid date string: ${dateCreated}`,
+    );
+    return null;
+  }
```

Comment:
We don't allow anything but a date to be entered in the data model, so I wonder if we can drop this check.

Reply:
Seems like adding `date_created: new Date().toISOString(),` to the `ENTITY` objects in both failing test files solves the problem.